index.html

```diff
@@ -20,20 +20,217 @@
             <div class="bot">
-            <input type="button" value="Start Collaboration ">
+                <button type="button" id="bom">Start Collaboration</button>
         </div>
-
     </section>
+    </header>
+
+    <main>    
     <div class="sectiontwo">
-        <!-- <img src="./asset/shape.svg" alt="A joined shape image"> -->
+        <img src="./asset/shape.svg" alt="">
+    </div>
+    <div class="pro">
+        <h1>projects</h1>
+    </div>
+    <div id="section-three">
+        <div class="lap">
+            <h1>welcome</h1>
+        <img src="./asset/laptop-image.png" alt="">
+    </div>
     </div>
+    <section class="section-plan">
+        <div class="plan">
+        <h4>My Project Portfolio</h4>
+        <ul id="ul">
+            <li>HTML/CSS</li>
+            <lI>Ruby on Rails</lI>
+            <li>Javascript</li>
+        </ul>
+            <div class="see">
+                <p>see this project <img src="./asset/arrowright.png" alt=""></p>
+            </div>
+        
+    </div>
+    </section>
+    <div id="section-threeb">
+        <div class="lap">
+        <img src="./asset/laptop2image.png" alt="">
+    </div>
+    </div>
+    <section class="section-plan">
+        <div class="plan">
+        <h4>My Project Portfolio</h4>
+        <ul id="ultw">
+            <li>HTML/CSS</li>
+            <lI>Ruby on Rails</lI>
+            <li>Javascript</li>
+        </ul>
+            <div class="see">
+                <p>see this project <img src="./asset/arrowright.png" alt=""></p>
+            </div>
+        
+    </div>
+    </section>
+<div id="section-threec">
+    <div class="lap">
+    <img src="./asset/laptopfour.png" alt="">
+</div>
+</div>
+<section class="section-plan">
+    <div class="plan">
+    <h4>My Project Portfolio</h4>
+    <ul id="ulth">
+        <li>HTML/CSS</li>
+        <lI>Ruby on Rails</lI>
+        <li>Javascript</li>
+    </ul>
+        <div class="see">
+            <p>see this project <img src="./asset/arrowright.png" alt=""></p>
+        </div>
+    
+</div>
+</section>
+<div id="section-threed">
+    <div class="lap">
+    <img src="./asset/laptopfive.png" alt="">
+</div>
+</div>
+<section class="section-plan">
+    <div class="plan">
+    <h4>My Project Portfolio</h4>
+    <ul id="ulfo">
+        <li>HTML/CSS</li>
+        <lI>Ruby on Rails</lI>
+        <li>Javascript</li>
+    </ul>
+        <div class="see">
+            <p>see this project <img src="./asset/arrowright.png" alt=""></p>
+        </div>
+    
+</div>
+</section>
+<div id="section-threee">
+    <div class="lap">
+    <img src="./asset/laptopsix.png" alt="">
+</div>
+</div>
+<section class="section-plan">
+    <div class="plan">
+    <h4>My Project Portfolio</h4>
+    <ul id="ulfi">
+        <li>HTML/CSS</li>
+        <lI>Ruby on Rails</lI>
+        <li>Javascript</li>
+    </ul>
+        <div class="see">
+            <p>see this project <img src="./asset/arrowright.png" alt=""></p>
+        </div>
```

Comment:
- [ ] It should be either a button tag or a tag so kindly make changes to it mentioned by the previous reviewer.